Array_Rotation/Array_Rotation.rb

```diff
@@ -0,0 +1,32 @@
+# Function for the Array Rotation
+def Left_Array_Rotation(arr , n)
+    temp = arr[0]
+    for i in 0..n-2
+        arr[i] = arr[i+1]
+    end
+    arr[n-1] = temp
+    return arr
+end
```

Comment:
Yes, please explain through comments what is left_array_rotation doing.
Purpose of using the function and so on.